opds_import.py

```diff
@@ -266,7 +266,7 @@ def import_from_feed(self, feed, even_if_no_author=False,
             except Exception, e:
                 identifier, ignore = Identifier.parse_urn(self._db, key)
                 data_source = DataSource.lookup(self._db, self.data_source_name)
-                failure = CoverageFailure(identifier, traceback.format_exc(), data_source=data_source, transient=False)
+                failure = CoverageFailure(identifier, traceback.format_exc(), data_source=data_source, transient=True)
```

Comment:
I don't think that would be too bad but it's true that my change was not terribly well thought out--there's code a little bit earlier that treats a similar exception as a transient error. So I've changed it back for the time being.